### DISCUSSION TITLE
Main language not in its own subfolder

### DISCUSSION THREAD

**Discussion by Because789** [sveltia/sveltia-cms#125] [Q&A] [answered]
Just to make sure, am I right that for now it's not possible to have the main language content of a collection out of its own subfolder? Or in other words, we can't, configure Sveltia to handle this folder structure:

```sh
blog/
  ├── en/
  │       ├── other-language-content.md
  ├── main-language-content.md
```
Right, or did I miss something?

**Comment by kyoshino** [comment, marked as answer]
Yeah, for now, it’s not possible to have such a structure in Sveltia CMS as we follow the [i18n support in Decap CMS](https://decapcms.org/docs/i18n/). I could add the support for it if there are specific needs.

**Reply by Because789** [reply]
Awesome, thank you! I'm not sure if it's needed, I'm still trying to find the best way to implement i18n in Astro to my needs, and that might include this structure, or it might not. Will come back to you, if I find an important use case...

While I'm here, thanks so much for your work on Sveltia! It's turning out really great, and I'm excited to see where you're going with it 🚀
